applications/service_brokers/installing-service-catalog.adoc

```diff
@@ -17,3 +17,20 @@ include::modules/modules/sb-about-service-catalog.adoc[leveloffset=+1]
 
 // Installing service catalog
 include::modules/modules/sb-install-service-catalog.adoc[leveloffset=+1]
+
+// Uninstalling service catalog
+include::modules/modules/sb-uninstall-service-catalog.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+There is a known issue related to projects getting stuck in a "Terminating" state when attempting to delete them after disabling the service catalog.
+ifdef::openshift-enterprise[]
+See
+xref:../../release_notes/ocp-4-1-release-notes.html#ocp-4-1-known-issues[Release
```

Comment:
Was going to suggest to put this note in the procedure (as opposed to seeing it after the procedure), but then I realized there was an xref... :(.

Reply:
:(